src/test/java/pages/LoginPage.java

```diff
@@ -38,8 +38,10 @@ public LoginPage providePassword(String password) {
         return this;
     }
 
-    public LoginPage logIn(){
-        provideEmail("demo@class.com");
+
+    public void logIn(){
+        provideEmail("terrence.regis@gmail.com");
+=
```

Comment:
You got this problem, please correct it. 


![image](https://user-images.githubusercontent.com/104761482/214798572-e2924eb8-8a75-4381-b18c-30683685db68.png)